lib/commands/build.ts

```diff
@@ -41,12 +42,19 @@ export abstract class BuildCommandBase extends ValidatePlatformCommandBase {
 			keyStoreAlias: this.$options.keyStoreAlias,
 			keyStorePath: this.$options.keyStorePath,
 			keyStoreAliasPassword: this.$options.keyStoreAliasPassword,
-			keyStorePassword: this.$options.keyStorePassword
+			keyStorePassword: this.$options.keyStorePassword,
+			androidBundle: this.$options.aab
 		};
-		await this.$platformService.buildPlatform(platform, buildConfig, this.$projectData);
+
+		const outputPath = await this.$platformService.buildPlatform(platform, buildConfig, this.$projectData);
+
 		if (this.$options.copyTo) {
 			this.$platformService.copyLastOutput(platform, this.$options.copyTo, buildConfig, this.$projectData);
+		} else {
+			this.$logger.info(`Your build result is located at: ${outputPath}`);
```

Comment:
I am not very familiar with the CLI errors, but if you/your is not used a lot, I would recommend changing this to "**The** build result is located at" instead of "Your..."